Localization of upstream and other stuff

ThxMqninq opened the pull request:
<!--
SPDX-FileCopyrightText: 2021 Pieter-Jan Briers <pieterjan.briers+git@gmail.com>
SPDX-FileCopyrightText: 2021 Swept <sweptwastaken@protonmail.com>
SPDX-FileCopyrightText: 2021 mirrorcult <lunarautomaton6@gmail.com>
SPDX-FileCopyrightText: 2022 AJCM-git <60196617+AJCM-git@users.noreply.github.com>
SPDX-FileCopyrightText: 2022 Kara <lunarautomaton6@gmail.com>
SPDX-FileCopyrightText: 2023 DrSmugleaf <DrSmugleaf@users.noreply.github.com>
SPDX-FileCopyrightText: 2023 Kevin Zheng <kevinz5000@gmail.com>
SPDX-FileCopyrightText: 2024 Vasilis <vasilis@pikachu.systems>
SPDX-FileCopyrightText: 2024 lzk <124214523+lzk228@users.noreply.github.com>
SPDX-FileCopyrightText: 2025 Aiden <28298836+Aidenkrz@users.noreply.github.com>

SPDX-License-Identifier: AGPL-3.0-or-later
-->

<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- NOTE: All code submitted to this repository is ALWAYS licensed under the AGPL-3.0-or-later license. 
The REUSE Specification headers or separate .license files indicate a secondary license (e.g., MPL or MIT) solely to facilitate 
integration for projects that do not use the AGPL license. This secondary license does not replace the fact that AGPL-3.0-or-later remains the primary and binding license. 
Uncomment and modify the following line if you wish to change the license from the default of AGPL.-->
<!--- LICENSE: AGPL -->
## About the PR
<!-- What did you change? -->
Локализация всякого с апстрима, исправления.

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl: Jose229
- add: Добавлена локализация!
- remove: Убрана локализация!
- tweak: Изменена локализация!
- fix: Исправлена локализация!

lexaSvarshik committed:
Update RobustToolbox

HATE IT

lexaSvarshik committed:
[Port] Custom announce (#227)

* announcecustom command (#2163)

* rushed af

* Update AnnounceCustomCommand.cs

* translations

---------

Co-authored-by: Milon <milonpl.git@proton.me>

lexaSvarshik committed:
New space law (#212)

* new law is here

* decolour some boxes

* fixes of other stuff

* better locale

* basically all here i guess

* prettyfing

lexaSvarshik committed:
[Port] Crime Assist pda program (#178)

* Crime Assist pda program (#464)

* Completely untested first commit

* Get basic program flow working

* Add Innocent

* Add Fancy Rich Text

* Icon

Thanks, IcedQuinn!

* Add Cartridge icons

* New image from IcedQuinn

* Add explanation on what a Sophont is, to be automatically appended to any text referencing a Sophont later on

* Convert tabs to spaces

* remove [bold] tags from loc

* Make text bold again

* Alter formatting, reduce text length for sophont explanation to fit it on the screen

* Update Resources/Locale/en-US/deltav/cartridge-loader/cartridges.ftl

Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Signed-off-by: Tim Falken <timfalken@hotmail.com>

* Add program to more PDA's

* Add a tip about the app

* Recreate flow in yaml

todo: actually load and use this yaml

* Remove the old state machine, and add configurable yaml prototypes to define pages

* Move page to its own file

---------

Signed-off-by: Tim Falken <timfalken@hotmail.com>
Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>

* Cassette-Cartridges (#2108)

* Hello cassette-tapes for mail, crimeassist and psionics chart app thing.

* Goodbye, version of this sprite in the nyanotrasen folder

* delete redundant things

* Space law adapted to Reserve. Better Ru locale

Изменил некоторые статьи, лишние удалил. Уточнил некоторые формулировки, переместил строки в файле для более удобной читаемости и навигации.

* namespace changes

* delete ERP

* no fucking clue why it were gone

* tweaks due to new space law

* localize + law tweaks

---------

Signed-off-by: Tim Falken <timfalken@hotmail.com>
Co-authored-by: Tim Falken <timfalken@hotmail.com>
Co-authored-by: Colin-Tel <113523727+Colin-Tel@users.noreply.github.com>
Co-authored-by: Monotheonist <182929384+Monotheonist@users.noreply.github.com>

lexaSvarshik committed:
More translations of upstream things yay (#226)

* translating shit

* and also a few translations

* pizza fix

lexaSvarshik committed:
Publish tweak (#231)

penis to cdn

lexaSvarshik committed:
automatic changelog update yay (#232)

ThxMqninq committed:
localization

ThxMqninq committed:
shitchap

ThxMqninq committed:
update, fix & localize _Lavaland

ThxMqninq committed:
localize & fix _NF

ThxMqninq committed:
localize _Floofstation

ThxMqninq committed:
localize _EinsteinEngines

lexaSvarshik committed:
[Port] Better blob facrory (#234)

* Factories can now make more Blob Pods when the Pods die (#565)

* markings

* fixes

---------

Co-authored-by: Skye <57879983+Rainbeon@users.noreply.github.com>

lexaSvarshik committed:
Revenant buff port (#138)

* damage attacker and stun them

При атаке ревенанта атакующий получает 10 урона холодом, и стан на 4 секунды.

* cold damage ignore resistanses, shock do 0

гарантированно наносит 10 холода, электрошок не наносит урона

* Add IgnoreInsulation vatiable

* ignore insuls  component add

+small tweak so it works just fine

* cold damage on contact

Когда ревенант в телесной форме, при контакте с ним получаете 5 урона холодом. Бойтесь ревы!!

* break lights via defile ability

Теперь ревенант не просто моргает лампочками, а бьет их!

* Update revenant.yml

check fix

* balanced structural damage

теперь ревенант ломает новое целое окно за 4 прожима абилки. Если окно было прежде хоть чуть повреждено, ломает за три

* Update RevenantSystem.Abilities.cs

* Ни слова про 220

* Удалил дублирующееся поле

* weak to holy added

* Add 3 abilities to revenant, salt weaknesses, and evil bible interaction (#3063)

* Added the blood writing ability to revenants

(cherry picked from commit 8157d155e1f7a3840992b54475c59031d2a75f25)

* Added the revenant Animate action

(cherry picked from commit 1b53520849a60538598cae641c6e598b1a5d7534)

* Added Animate action sprite and fixed revenant action sprites being small

(cherry picked from commit 5c88ce9b21cf517023cd136ddb8aaa361f72fbfe)

* Revenants in stasis can now be crafted into sentient revenant plushies

Revenants in stasis can now talk

(cherry picked from commit af84083da6e17bb7671d9f00993dc6bcc0b0a583)

* Revenant plushies and ectoplasm now talk with their trapped revenant's name

(cherry picked from commit 855b25aeb7f29ed6ff2dcc0d4ab1e589ae7e2e16)

* Added a yml nerf option in case revenant is too powerful

(cherry picked from commit 6895a5ca929f32267dea1e326aee62302dc151b8)

* Added a yml nerf option in case revenant is too powerful

(cherry picked from commit 6895a5ca929f32267dea1e326aee62302dc151b8)

* Passing over salt ore now reveals revenants

(cherry picked from commit beea55885f98adc72e34751ba6f798bfadd8dd31)

* Salt puddles now reveal revenants

Hooray for EnsureTileReaction!

(cherry picked from commit b69304747e3b5aa867c5afd8917d2c170c65253e)

* Bibles now reveal revenants

Localization strings added

(cherry picked from commit 62f7ec604748697e5fecc942c4a5d14ff682c596)

* Added the revenant Haunt action

(cherry picked from commit fb692c4cab3f411c11a3004fa73f30122e0ff7f9)

* Haunt witnesses now have an icon over their head on the revenant's client

(cherry picked from commit 5d1b3c6b0af4508ebee7aa8738f7a5f3aa9e53ca)

* Slight revenant nerfs and an additional escape hatch

(cherry picked from commit 490e4c525b75b03502497bfa89827f915f6c7fb2)

* Fixed revenant speech name, and other fixes

This is a few changes clumped into one. I should've broken this one up, but I got overzealous.
- Revenant's ectoplasm now properly uses TransformSpeakerNameEvent
- Revenants crafted into plushies now use VoiceOverrideComponent instead of VoiceMaskComponent
- Plushies now have a custom speaking noise
- Moved Impstation-specific revenant code to the _Impstation subdirectory

(cherry picked from commit a5e0b0c9383e4011e69edc0592c8a4cf40b06402)

* Fixed revenant haunt witness sprite not disappearing

(cherry picked from commit 0c06d4a8f48f6eee324cd944dbcb1fce3258eb58)

* New artifact effectr: animate nearby items (like a revenant)

(cherry picked from commit 6ba1b4d8b313eda9e5ff82246ed7425e80287d0f)

* Revenant cleanups

* Revenant Fixes

* add audio

* haunt fixes

* salt fixes

* grinder gamer

* stasis

* cleanup and comments

* deltav ectoplasm

* clean up imp comment

* event weight and guidebook gaming

* catalog

* alert fix

* window nerf

* effectprobfake

* mapinit

* no braincells ops

Signed-off-by: Lyndomen <49795619+Lyndomen@users.noreply.github.com>

* add physics

* remove phyiscs

* add physics realer

* bible

---------

Signed-off-by: Lyndomen <49795619+Lyndomen@users.noreply.github.com>
Co-authored-by: TGRCDev <tgrc@tgrc.dev>
Co-authored-by: TheGrimbeeper <thegrimbeeper.11@gmail.com>

* translations and fixes

* aghost doesnt push revenant

* sounds for all abitilies and proper ftl

* Merge pull request #247 from TGRCdev/revenant-animate-action

Added the revenant Animate action

* duplicates removed

* fixed bugs

Исправил баг, когда двуручное оружие не могло стрелять из-за щиткода губов.

---------

Signed-off-by: Lyndomen <49795619+Lyndomen@users.noreply.github.com>
Co-authored-by: Кирилл Хилювчиц <kirus0059@gmail.com>
Co-authored-by: Lyndomen <49795619+Lyndomen@users.noreply.github.com>
Co-authored-by: TGRCDev <tgrc@tgrc.dev>
Co-authored-by: TheGrimbeeper <thegrimbeeper.11@gmail.com>
Co-authored-by: V <97265903+formlessnameless@users.noreply.github.com>

ThxMqninq committed:
update & fix _DV

ThxMqninq committed:
localize _Goobstation part 1

lexaSvarshik commented:
![livrah-ss13](https://github.com/user-attachments/assets/84fba6d7-fed5-4ce8-a637-6a6a1642de95)

ThxMqninq committed:
cosmic cult localize & fix

ThxMqninq committed:
localize _Shitmed and woundmed part 1

lexaSvarshik committed:
[Port] Vox raiders (#235)

* Merge pull request #286 from FireNameFN/VoxRaiders

Воксы-налётчики

* vox walls (#287)

* Merge pull request #291 from FireNameFN/VoxRaiders

vox-raiders part 2

* Update VerbCategory.cs

Вернул прикол который был вырезан

* Shipping Containers (#731)

Co-authored-by: lapatison <100279397+lapatison@users.noreply.github.com>

* small fixes

* Concrete fix again (#2488)

* baunch of fixes

---------

Co-authored-by: Ko4ergaPunk <kunjutnitsa@gmail.com>
Co-authored-by: ksen0morph <144564556+ksen0morph@users.noreply.github.com>
Co-authored-by: Morb <14136326+Morb0@users.noreply.github.com>
Co-authored-by: lapatison <100279397+lapatison@users.noreply.github.com>
Co-authored-by: NotSoDamn <75203942+NotSoDana@users.noreply.github.com>

lexaSvarshik committed:
[Port] Battle royale (#185)

* BattleRoyaleMod (#325)

Co-authored-by: FN <37689533+FireNameFN@users.noreply.github.com>

* Battle Royal Map (#330)

* Фиксы Королевской Битвы (#333)

* BattleRoyaleFixes

* BattleRoyaleFixes

* BattleRoyaleFixes

* FixAttributes

* BattleRoyaleFixes

* GameRule fixes

Фиксы геймрула, немпспейсы и прочее

* Concrete fix again (#2488)

* вернул как было

* goobmaths

* map fixes

* Retexture grass & dirt tiles (#2282)

* fixed grass (maybe passenger)

* [Resprite + Suit] CBURN and CBURN commander (#662)

* bags fix

* moved map to corvax dir

* final fixes

* rsi fixes

---------

Co-authored-by: Kill_Me_I_Noobs <118206719+Vonsant@users.noreply.github.com>
Co-authored-by: FN <37689533+FireNameFN@users.noreply.github.com>
Co-authored-by: NotSoDamn <75203942+NotSoDana@users.noreply.github.com>
Co-authored-by: Ko4ergaPunk <62609550+Ko4ergaPunk@users.noreply.github.com>
Co-authored-by: no_mad <110369449+nomad0260@users.noreply.github.com>

lexaSvarshik committed:
small tweaks dont care about em (#237)

сачтук

lexaSvarshik committed:
Revenant fixes + monument visibiluty (#238)

* vis flags and rev fix 1

* fixed stasis

---------

Co-authored-by: beck <163376292+widgetbeck@users.noreply.github.com>

ThxMqninq committed:
_Shitmed part 2

lexaSvarshik committed:
Shotgun more sounds (#236)

Co-authored-by: Morb <14136326+Morb0@users.noreply.github.com>

ThxMqninq committed:
_Goobstation melee weapons localize

lexaSvarshik committed:
Update server-updates.ftl

funny text

ThxMqninq committed:
_Goobstation guns

ThxMqninq committed:
fix locale error in Content.Server

ThxMqninq committed:
1 file ss-14

ThxMqninq committed:
_Goobstation part 2

ThxMqninq committed:
cosmic cult structures localization & fix

lexaSvarshik commented:
Конфликты

ca4tuk committed:
Update ServerApi.cs

lexaSvarshik commented:
куда нахуй

lexaSvarshik committed:
Revert "merge master"

This reverts commit 928ec93441c38f72afea7f7d888d7ff902afb8ed, reversing
changes made to 65f92c014a59615d93d2b15b3bcbb62a28acbec0.

lexaSvarshik committed:
Reapply "merge master"

This reverts commit b8912e008347dbe981b3439db8c642286b9c81cd.

lexaSvarshik commented:
ДА Я ЕБУ

lexaSvarshik commented:
Перенесено
